src/pages/Appointments/AppointmentsPage.tsx

```
@@ -107,19 +107,30 @@ function DateRangeDisplay({ dateFrom, dateTo }: DateRangeDisplayProps) {
+            ({formatDate(dateFrom, "dd MMM yyyy")})
+          </span>
+        </>
+      ) : (
+        <>
+          <span className="text-black">{t("yesterday")}</span>
+          <span className="pl-1 text-gray-500">
+            ({formatDate(dateFrom, "dd MMM yyyy")})
+          </span>
+        </>
+      )}
+    </>;
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)